frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -41,6 +40,11 @@ local ReaderHighlight = InputContainer:extend{
     },
 }
 
+function ReaderHighlight:getHighlightColorHash(color)
+    return Blitbuffer.colorFromName(color) --- @todo fix in the night mode; issue #14170, #14667
+        or Blitbuffer.gray(self.view.highlight.lighten_factor) -- 'gray' or unknown color
+end
```

Comment:
It's possible I see too much of the word "hash" in crengine and at work, and it evokes for me https://en.wikipedia.org/wiki/Hash_function (and not being young, it doesn't evoke "hash tag koreader" at all :)).
I think here, it returns the hex value (#BB4499) of that color ?
https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Values/color_value calls it "hex color".
So, maybe `getHighlightColorAsHexColor` or `getHighlightHexColor` ?

Reply:
It returns ColorRGB32 or Color8 structure.
Isn't it a "hash table"?